### DISCUSSION TITLE
New Install has missing info and Errors in kibana dashboards.

### DISCUSSION THREAD

**Discussion by charlespctech** [Security-Onion-Solutions/securityonion#7612] [Unsupported Versions] [answered]
I am am seeing multiple dashboards in kibana showing a red error button. They all say stuff similar to the following : 

"Error [esaags ]> Saved field "source.ip" of index patten "*:so-" is invald with the "terms" aggregation. Please select a new field"

This is a brand new install. 

Everything else seems to be working however I am unable to see some useful built-in dashboards such as number of source ip addresses on the network.

**Comment by charlespctech** [comment]
![image](https://user-images.githubusercontent.com/7142801/159186222-ff45ce1a-b221-4b5b-a702-068cb0e9eee7.png)

**Comment by charlespctech** [comment]
When I scroll down I do see logs that have entries for the source and destination ip's so its strange that they aren't populating above.
![image](https://user-images.githubusercontent.com/7142801/159186359-527bd55d-ca2e-4f9a-a848-0569f9d6589d.png)

**Comment by dougburks** [comment, marked as answer]
What options did you choose when you ran Setup?

Were there any errors during Setup?

Are you just consuming network traffic or are you sending any additional logs?

What do you see if you navigate to `Stack Management` -> `Index Patterns` -> `*:so-*` in Kibana?

What happens if you use our [Hunt](https://docs.securityonion.net/en/2.3/hunt.html) interface and specify a query like `* | groupby source.ip destination.ip`?

**Reply by charlespctech** [reply]
> What options did you choose when you ran Setup?

Eval version, dhcp then changed to static, all modules enabled

> Were there any errors during Setup?

Yes but I thought they were due to the DNS. Once I setup my routers dns resolver and created a dns record for that host the installation completed successfully and I was allowed to access the web interface.

> Are you just consuming network traffic or are you sending any additional logs?

So far just network traffic. I did install 1 instance of os query though

> What do you see if you navigate to `Stack Management` -> `Index Patterns` -> `*:so-*` in Kibana?

I see both source.ip and destination.ip as showing not aggregable
![image](https://user-images.githubusercontent.com/7142801/159363578-77547825-bee4-441f-b1c6-beb5fc81de8b.png)


> What happens if you use our [Hunt](https://docs.securityonion.net/en/2.3/hunt.html) interface and specify a query like `* | groupby source.ip destination.ip`?

I see a large amount of results

![image](https://user-images.githubusercontent.com/7142801/159364073-5f45da9a-ea48-48e5-85e8-96d3115447c3.png)

**Reply by dougburks** [reply]
source.ip and destination.ip should be marked as aggregatable:
![image](https://user-images.githubusercontent.com/1659467/159685546-7125b782-610f-40fc-a3e5-009d152e4af7.png)

You could try reloading the Kibana config:
```
sudo rm /opt/so/state/kibana_*.txt
sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True
```

If that doesn't help, you might want to go ahead and reinstall since your initial installation had errors.
